src/app/views/realms/view/view.store.ts

```diff
@@ -0,0 +1,42 @@
+import { inject, InjectionKey, ref, Ref } from 'vue';
+import { Fireside } from '../../../../_common/fireside/fireside.model';
+import { ModelClassType } from '../../../../_common/model/model.service';
+import { Realm } from '../../../../_common/realm/realm-model';
+import { User } from '../../../../_common/user/user.model';
+
+export const RealmRouteStoreKey: InjectionKey<RealmRouteStore> = Symbol('realm-store');
+
+export type RealmRouteStore = ReturnType<typeof createRealmRouteStore>;
+
+export type RealmRoutePayload = {
+	realm: ModelClassType<Realm>;
+	knownFollowers: ModelClassType<User>[];
+	knownFollowerCount: number;
+	activeFiresides: ModelClassType<Fireside>[];
+};
+
+export function createRealmRouteStore() {
+	const realm = ref() as Ref<Realm>;
+	const knownFollowers = ref<User[]>([]);
+	const knownFollowerCount = ref(0);
+	const firesides = ref<Fireside[]>([]);
```

Comment:
I think that we should also have `userFireside` here. Community route was getting both some preview firesides along with our own fireside, so we would replace the FiresideAdd component with our own fireside if it existed. Check `src/app/views/communities/view/overview/overview.vue` for how we handled the firesides there.

Reply:
hey, maybe im looking in the wrong place but i don't see it being used.
its used when you fetch the home route but doesnt seem to show on the community itself.
whats the intended behaviour for this?